src/sentry/snuba/metrics/query_builder.py

```diff
@@ -149,7 +154,7 @@ def parse_query(query_string: str) -> Sequence[Condition]:
     return where
 
 
-class QueryDefinition:
+class APIQueryDefinition:
```

Comment:
I would like to move this class to `src/sentry/api/endpoints/organization_metrics.py` in a follow-up PR, to create a clearer separation between web API and programming interface.